Composer/cypress/support/commands.ts

```diff
@@ -19,6 +21,65 @@ Cypress.Commands.add('createBot', (botId: string, botName?: string) => {
   });
 });
 
+Cypress.Commands.add('createBotV2', (botName: string, callback: (bot: any) => void) => {
+  const params = {
+    description: '',
+    location: '',
+    name: botName,
+    runtimeLanguage: 'dotnet',
+    runtimeType: 'webapp',
+    schemaUrl: '',
+    storageId: 'default',
+    templateId: '@microsoft/generator-bot-empty',
+    templateVersion: '1.0.0-rc3',
+  };
+
+  const pollingRequestBotStatus = (jobId: string, callback: (result: any) => void) => {
+    cy.wait(2000);
+    try {
+      cy.request('get', `/api/status/${jobId}`).then((res) => {
+        const { httpStatusCode, id, result } = res.body;
+        if (httpStatusCode !== 200) {
+          pollingRequestBotStatus(id, callback);
+        } else {
+          callback(result);
+        }
+      });
+    } catch (error) {
+      pollingRequestBotStatus(jobId, callback);
+    }
+  };
+
+  cy.request('post', '/api/v2/projects', params).then((res) => {
+    const { jobId } = res.body;
+    // install package can take a long time.
+    cy.wait(20000);
+    pollingRequestBotStatus(jobId, (result) => callback(result));
+  });
+});
+
+Cypress.Commands.add('createTemplateBot', (botName: string, callback: (bot: any) => void) => {
+  cy.createBotV2(`TemplateBot_${botName}`, (bot) => {
+    TemplateBotProjectId = bot.id;
+  });
+});
+
+Cypress.Commands.add('createTestBot', (botName: string, callback: (bot: any) => void) => {
+  const name = `TestBot_${botName}`;
```

Comment:
name not prefix with `__Test*`  anymore is to meet our server name validation.  https://github.com/microsoft/BotFramework-Composer/blob/33f63dbda4d8c67d1f0095314999abb6e65b21d9/Composer/packages/lib/shared/src/dialogUtils/validateDialogName.ts#L7